components/chart/linearTimeSeries.js

```diff
@@ -0,0 +1,127 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.charts.linearTimeSeries = function() {
+
+        var xScale = d3.time.scale();
+        var yScale = d3.scale.linear();
+        var xAxis = d3.svg.axis()
+            .scale(xScale)
+            .orient('bottom');
+        var yAxis = d3.svg.axis()
+            .scale(yScale)
+            .orient('left');
```

Comment:
OK, I'm happy either way. The fact that it is baked-in isn't an issue in my opinion.

We really should promote the fact that the components are small, and encourage copy-paste-modify as a way of using this code.